contracts/libraries/PeripherySigningLib.sol

```diff
@@ -78,6 +83,7 @@ library PeripherySigningLib {
             keccak256(
                 abi.encode(
                     EIP712_BASE_DEPOSIT_DATA_TYPEHASH,
+                    baseDepositData.inputToken,
```

Comment:
I think we were missing this in this typehash

Reply:
Ah we definitely were.